app/timetables/models.py

```diff
@@ -37,8 +37,10 @@ class Meal(SlugifyMixin, models.Model):
     slugify_field = 'name'
 
     def clean(self):
-        if self.start_time >= self.end_time:
-            raise ValidationError(_('start_time must be less than end_time.'))
+        # Ensure start time and end time are not None before compare
+        if self.start_time and self.end_time:
+            if self.start_time >= self.end_time:
```

Comment:
Flat is better than nested (Zen of Python). @andela-kanyanwu I think that one-liner is clear enough though nested `if` shouldn't be what's stopping this PR.
@andela-joyebanji Simple is better than complex (Zen of Python). I think the current implementation isn't complex.